README.md

```diff
@@ -47,14 +47,14 @@ We are a new and growing project and welcome [contributions](CONTRIBUTING.md).
 To install the latest official release:
 
 ```
-pip install keras-nlp tensorflow --upgrade
```

Comment:
Why don't we need to upgrade anymore?

Reply:
Hmm, yeah let me leave the `--upgrade`. It is no longer needed to update TensorFlow as of https://github.com/keras-team/keras-nlp/pull/697, but useful to make sure you bump your keras-nlp install to the latest.